boards/FSM/Src/main.c

```diff
@@ -173,8 +174,12 @@ int main(void)
     MX_CAN_Init();
     MX_IWDG_Init();
     MX_ADC2_Init();
+    MX_TIM4_Init();
     /* USER CODE BEGIN 2 */
-    primary_flow_meter = App_FlowMeter_Create(Io_FlowMeter_GetPrimaryFlowRate);
```

Comment:
This block of code should be between `/* USER CODE BEGIN 1 */` and `/* USER CODE END 1 */` because we want to register flow meters to FsmWorld in a later PR.

Reply:
moved to user code begin 1